package.json

```diff
@@ -1,22 +1,19 @@
 {
   "name": "unleash-proxy-client",
-  "version": "2.5.0",
+  "version": "2.5.1-beta.0",
   "description": "A browser client that can be used together with the unleash-proxy.",
-  "main": "./build/index.js",
-  "types": "./build/index.d.ts",
+  "main": "./build/cjs/index.js",
+  "types": "./build/cjs/index.d.ts",
   "files": [
     "build",
     "examples",
     "src"
   ],
   "scripts": {
-    "preversion": "rm -rf build && npm run build",
-    "prepare": "npm run build && ./scripts/check-release.sh",
-    "build": "npm run build:ts && npm run build:web",
-    "build:ts": "tsc",
-    "build:web": "rollup -c rollup.config.js",
-    "fmt": "prettier src --write --loglevel warn",
-    "lint": "eslint src",
+    "build": "rm -rf build && tsc && rollup -c rollup.config.mjs",
```

Comment:
Nit: Here the build is doing a clean & build, which seems like two different steps, but for CI/CD it doesn't matter much because build folder will not exist

Reply:
`rm -f` will not cause an error if the target doesn't exist